whatis277/harvest
# bean: Update go to 1.22.0

Updates go version and dependencies

The primary reason behind this upgrade is the new router ([ref](https://eli.thegreenplace.net/2023/better-http-server-routing-in-go-122/))

Testing instructions:
1. `dc up --build bean`
2. Check logs to ensure nothing is wrong
3. `dc exec -e INTEGRATION=1 bean go test ./...`
4. Go to http://localhost:8080/
5. Ensure it's working
